File: 8kyu 2/repeatStr.js
/*
|--------------------------------------------------------------------------
| String repeat
|--------------------------------------------------------------------------
|
*/

function repeatStr(n, s) {
   return s.repeat(n);
}


const repeatStr = (n, s) => s.repeat(n);


function repeatStr(n, s) {
   var str = "";
   for (var i = 0; i < n; i++)
      str += s;
   return str;
}


console.log(repeatStr(3, "*"), "***");
console.log(repeatStr(5, "#"), "#####");
console.log(repeatStr(2, "ha "), "ha ha ");

File: Python/8py 3/repeat_str.py
"""
|--------------------------------------------------------------------------
| String repeat
|--------------------------------------------------------------------------
|
"""

def repeat_str(repeat, string):
   return string*repeat


repeat_str = lambda r, s: s * r


def repeat_str(repeat, string):
   solution = ""
   for i in range(repeat):
      solution += string
   return solution

   
def repeat_str(repeat, string):
   return "".join([string]*repeat)
   
   
from operator import mul as repeat_str


print(repeat_str(4, 'a'), 'aaaa')
print(repeat_str(3, 'hello '), 'hello hello hello ')
print(repeat_str(2, 'abc'), 'abcabc')